chapters/extensions/VK_KHR_synchronization2.md

```diff
@@ -0,0 +1,293 @@
+# VK_KHR_synchronization2
+
+The following information was written with the goal of helpping guide developers wanting to update to using `VK_KHR_synchronization2` where possible. It will also give some examples of how the original Vulkan synchronization is different.
```

Comment:
```suggestion
The following information was written with the goal of helping guide developers wanting to update to using `VK_KHR_synchronization2` where possible. It will also give some examples of how the original Vulkan synchronization is different.
```